torch/_refs/nn/functional/__init__.py

```diff
@@ -116,7 +125,7 @@ def alpha_dropout(
     return self * dropout_mask + b
 
 
-def inplace_wrapper(fn):
+def _inplace_wrapper(fn):
```

Comment:
We already have `out_wrapper` without an initial underscore, so this one can probably be without an underscore as well.

Reply:
I think a closer comparison is to `_make_inplace` under `torch._refs.__init__.py`. IMO both are module specific helper function so I'd prefer not exposing publicly. wdyt?